packages/angular/src/generators/ng-add/migrators/migrator.ts

```diff
@@ -97,8 +97,7 @@ export abstract class Migrator {
 
     const nxJson = readNxJson(this.tree);
 
-    Object.keys(nxJson.tasksRunnerOptions ?? {}).forEach((taskRunnerName) => {
-      const taskRunner = nxJson.tasksRunnerOptions[taskRunnerName];
+    Object.values(nxJson.tasksRunnerOptions ?? {}).forEach((taskRunner) => {
```

Comment:
If I'm understanding the changes correctly, this shouldn't be needed anymore. This migrates an Angular CLI workspace to an Nx workspace. So, we shouldn't have `tasksRunnerOptions`.